doc/platformsh/README.md

```diff
@@ -1,4 +1,4 @@
-# Use eZ Platform on Platform.sh
+# Use eZ Platform Enterprise Edition on Platform.sh
```

Comment:
could we skip this diff, the file does not speak specifically abut ee stuff, so just want to keep future merge conflicts down to minimal if possible.